src/styles/index.css

```diff
@@ -20,3 +38,37 @@ header .buttons-become_collab {
 	background-color: white;
 	border-bottom: solid 1px black;
 }
+
+footer {
```

Comment:
Instead of using a tag selector, use a class selector
```suggestion
.footer {
```